pkg/kubelet/dockershim/security_context.go

```diff
@@ -113,6 +113,14 @@ func modifyHostConfig(sc *runtimeapi.LinuxContainerSecurityContext, hostConfig *
 	}
 	hostConfig.SecurityOpt = append(hostConfig.SecurityOpt, apparmorSecurityOpts...)
 
+	if sc.NoNewPrivs {
+		// "no-new-privileges" was added in Docker 1.11
+		// which is API version 1.23
+		if apiVersion.GTE(semver.Version{Major: 1, Minor: 23}) {
```

Comment:
Should we block the pod from running if `no-new-privileges` can't be enforced? That's what we do for apparmor.  (https://github.com/kubernetes/kubernetes/blob/3fb2d8248d7f367758292ac0964cf0d33a8f4678/pkg/kubelet/kubelet.go#L812-L813)